utils/test_memcheck.supp

```diff
@@ -504,16 +506,65 @@
 {
    <insert_a_suppression_name_here>
    Memcheck:Leak
-   match-leak-kinds:possible
+   ...
    fun:calloc
-   fun:lrua_array_alloc_one
-   fun:lrua_array_alloc
-   fun:vos_ts_table_alloc
-   fun:alloc_ts_cache
-   fun:ts_test_init
-   obj:/usr/lib64/libcmocka.so.*
+   fun:vea_reserve
+   fun:ut_*
+   obj:/usr/lib64/libcmocka.so*
+   fun:_cmocka_run_group_tests
+   fun:main
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Leak
+   match-leak-kinds:definite
+   fun:calloc
+   fun:eq_test_8
+   fun:main
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Leak
+   match-leak-kinds:indirect
+   fun:malloc
+   fun:do_alloc
```

Comment:
Outside the scope of this PR, but the do_alloc() function is our wrapper around malloc, so this issue and the one below it (which presumably represent regressions introduced since the memcheck tests were disabled - and will need fixing) could also be found by doing log analysis without the overhead of running valgrind, so in the normal Unit Test job, with very little additional overhead.  This would also give us the advantage of having a log file showing where the error occurs, although I suspect for these two the drpc stack trace will make it reasonably clear.

Reply:
memcheck tests were never "disabled" were they?   I thought the issue was that they were never truly enabled because the valgrind options were in the wrong code path in run_test.sh.  Only the tests that ran under subscripts (like btree.sh or evt_ctl.sh) were doing valgrind whereas everything else was just running without it.

Reply:
We weren't running some of the tests we thought we were.

Reply:
Correct, run_test.sh was not running valgrind for tests other than btree and evt_ctl.